### ISSUE TITLE
class path not recognised in cluster

### ISSUE THREAD

**Issue by shan1525** [brinckmann/montepython_public#231]
Hi everyone , 
I hope everyone is doing good. My system was taking so much time to run the MontePython , so I moved to run it in cluster . I installed class,planck likelihood, and montepython there. I checked if class is running or not, same thing with the planck likelihood. They are running well seperately.  But When I try to run the monetpython with the default.conf which I modified as suggested , which is as below ; 
root = '/mnt/home/associate/csomasri/shaz/MontePython_cluster'

path['cosmo']           = root+'/class_public-3.01/'
path['clik']            = root+'/code/plc_3.0/plc-3.1/'    

But when I run this in the cluster it is saying that file was not found for class. I am attaching the error below;
[error.txt](https://github.com/brinckmann/montepython_public/files/6998361/error.txt). 

I am running in the cluster with this  .sh file( I had to rename it because it was not uploading) below ;
[cluster_monte.txt](https://github.com/brinckmann/montepython_public/files/6998430/cluster_monte.txt)

If somebody has any idea or suggestion, Please share. It would really be helpful to me. 

Thanks in advance,
with the best regards,
shan.

**Comment by brinckmann** [comment]
It would seem you're giving it the wrong path to class, so you should check the path to your class directory and update it accordingly in the default.conf file. I can't help with that, except to say that it's looking in /mnt/home/associate/csomasri/shaz/MontePython_cluster/class_public-3.01/ which and it would be unusual to install class within a directory named after MontePython, but your mileage may differ.

Best,
Thejs

**Comment by shan1525** [comment]
Thank you so much @brinckmann  for your prompt and kind reply. I have installed all the files i.e. the montepython , class and planck likelihood folder in Montepython_cluster directory, should I rename it to something else? I will do that as you suggested. I don't know why it is not finding the path of class as I obtained the path using the pwd command. Then I copy-pasted it in the default.conf. I did the same thing in my system which I was running. For the cluster part I read the instructions given in the Boris_Bolliet page of github where he has mentioned it in the research.html file in the attachment folder of iomaster. May be I should look more to have a grasp about the error. I will right back if I find any clue . Thanks again for your reply. 

With the best regards, 
shan.

**Comment by shan1525** [comment]
Hi @brinckmann , with  your help and motivation I was able to solve the problem. As you mentioned that montepython was not finding the path for class and you told me to look carefully in my script. It turned out that as in planck likelihood we source the clik_profile.sh , similarly I had to add this line in my cluster_monte.sh file as this :
 source /mnt/home/associate/csomasri/.bash_profile . 
After doing this the montepython is running fine. Thanks for replying to our queries. It helps a lot. I am closing the issue as it has been resolved. 

with best regards,
shan
